lib/hsa/kernels/sum_hfb_wrapper.c

```diff
@@ -0,0 +1,765 @@
+//  gcc -std=gnu99 -O3 -Wall sum_hfb_wrapper.c -I$AMDAPPSDKROOT/include -lOpenCL -lm -L$AMDAPPSDKROOT/lib/x86_64/ -o sum_hfb_wrapper
```

Comment:
I'm not sure it makes sense to commit this wrapper here...  Can we drop this from the PR for now?